lib/OpenQA/Worker/Job.pm

```diff
@@ -479,16 +479,16 @@ sub _stop_step_5_2_upload {
 
     # do final status upload for selected reasons
     if ($reason eq WORKER_COMMAND_OBSOLETE) {
-        log_debug("Setting job $job_id to incomplete (obsolete)");
+        log_debug("Considering job $job_id as incomplete due to obsoletion");
```

Comment:
This doesn't seem to be covered by the unit tests? New or existing

Reply:
I've seen the diff in codecov. Considering how trivial the uncovered changes are I thought it wasn't worth adding tests.

Reply:
I have 100% statement coverage as a goal so ... it's worth it :)

Reply:
Of course there are always alternatives, e.g. make it a single-line statement or just get rid of code :)

Reply:
> I've seen the diff in codecov. Considering how trivial the uncovered changes are I thought it wasn't worth adding tests.

If none of the code paths is covered, I find it concerning. It's not just the log message. That just makes it more obvious.

Reply:
> It's not just the log message.

I know, hence my phrasing "trivial … changes".